src/Glpi/Controller/Form/TagListController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
```

Comment:
No need to add them in new files based on pieces of code that was produced after 2014.
```suggestion
```